content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md

```diff
@@ -183,6 +183,43 @@ There are some implicit conventions worth noting here:
     and you know that "zoneC" must be excluded. In this case, you can compose the yaml as below, so that "mypod" will be placed onto "zoneB" instead of "zoneC". Similarly `spec.nodeSelector` is also respected.
 
     {{< codenew file="pods/topology-spread-constraints/one-constraint-with-nodeaffinity.yaml" >}}
+    
+### Cluster-level default constraints
```

Comment:
Should we highlight it as "soft" default constraints?

Reply:
It is both